portal/views/portal.py

```diff
@@ -779,14 +786,16 @@ def settings():
         patient = get_user_or_abort(form.patient_id.data)
         try:
             dt = FHIR_datetime.parse(form.timestamp.data)
-            for qnr in QuestionnaireResponse.query.filter_by(subject_id=patient.id):
+            for qnr in QuestionnaireResponse.query.filter_by(
+                    subject_id=patient.id):
                 qnr.authored = dt
                 document = qnr.document
                 document['authored'] = FHIR_datetime.as_fhir(dt)
                 # Due to the infancy of JSON support in POSTGRES and SQLAlchemy
                 # one must force the update to get a JSON field change to stick
                 db.session.query(QuestionnaireResponse).filter(
-                    QuestionnaireResponse.id == qnr.id).update({"document": document})
+                    QuestionnaireResponse.id == qnr.id).update(
```

Comment:
Can you add a newline and put the `update()` part on its own line? (trying to keep the methods grouped with their options)